### DISCUSSION TITLE
DDP + Atmos

### DISCUSSION THREAD

**Discussion by Crosis47** [R3S3t9999/DoVi_Scripts#56] [Q&A]
Is the script capable of converting the TrueHD+Atmos track to DDP+Atmos? So far I've only been able to convert to a 7.1 EAC3 track and it drops the Atmos.

**Comment by R3S3t9999** [comment]
No sorry, the script cannot keep the Atmos metadata. There's a way but I don't like the quality, it messes up the channels volume.

**Comment by Crosis47** [comment]
In your experience, in regard to perceived quality, have you found it worth encoding the TrueHD+ATMOS  to 7.1 EAC3 or is it better to just use the studio core AC3 tracks? Just looking for some opinions beyond my own.

**Comment by R3S3t9999** [comment]
If you have a 7.1 setup yes.
if not, you won't notice much difference between a 1024kbps EAC3 file and a 640/448kbps AC3 core

**Comment by Crosis47** [comment]
I have an ATMOS-enabled Samsung Q80r soundbar with rear satellites which is technically a 7.1.2 setup so I guess it would make a difference since it is capable of decoding the 7.1 channels and outputting the audio. Thanks for the input. I'll start using the 7.1 re-encodes.

Which also reminds me, any idea why I'm getting these messages or is it normal?

![image](https://github.com/user-attachments/assets/e08e06ca-dac8-461b-95ef-c85f2f5a5bd8)

![image](https://github.com/user-attachments/assets/7efcbb70-c656-43ad-8712-26573f30a00b)

**Comment by R3S3t9999** [comment]
Yes, these msg are normal and don't matter.  Just don't close any window before the process has been completed
